rviz_rendering/include/rviz_rendering/objects/point_cloud_renderable.hpp

```diff
@@ -78,8 +78,17 @@ class PointCloudRenderable : public Ogre::SimpleRenderable
 #pragma clang diagnostic ignored "-Woverloaded-virtual"
 #endif
 
+// TODO(ivanpauno): Fixme!
+// This is not creating a problem, but hidding a base class virtual method is not a good practice.
```

Comment:
I can't remember why this was not fixed, but I remember something was complicating the change...

Reply:
It seems actually to be a false positive https://www.ogre3d.org/docs/api/1.7/class_ogre_1_1_simple_renderable.html#a51c5ae135793c602557d46be10c59b8d, as this doesn't seem to be hiding a base class method.

I can delete the TODO.
Besides the false positive, I still prefer adding the new compile option and explicitly ignoring it where it generates a problem.

Reply:
Yeah, that's fine, we're already suppressing the clang version of the warning.

Reply:
> Yeah, that's fine, we're already suppressing the clang version of the warning.

I didn't see it before :joy:.
I've deleted the directive for clang, as `GCC diagnostic` works for both https://github.com/ros2/rviz/pull/553/commits/32a232461d2d2e7debabcd39eebb2c226d4c5b98.
I've cleaned up the TODO too.